sdks/python/apache_beam/runners/portability/job_server.py

```diff
@@ -154,6 +158,78 @@ def local_temp_dir(self, **kwargs):
     return tempfile.mkdtemp(dir=self._local_temp_root, **kwargs)
 
 
+class JavaJarJobServer(SubprocessJobServer):
+
+  MAVEN_REPOSITORY = 'https://repo.maven.apache.org/maven2/org/apache/beam'
+  JAR_CACHE = os.path.expanduser("~/.apache_beam/cache")
+
```

Comment:
Not sure, but do we have to override `__init__()` here to call the super constructor?

Reply:
The super will automatically be called here, just like any other non-overridden method in the superclass.